src/main/java/com/ksm/wstbackoffice/domain/Warehouse.java

```diff
@@ -0,0 +1,27 @@
+package com.ksm.wstbackoffice.domain;
+
+import com.ksm.wstbackoffice.enums.PriceType;
+import com.ksm.wstbackoffice.enums.WarehouseType;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
```

Comment:
Please think about the need for Equals and HashCode for the warehouse.
Here is the explanation why: https://stackoverflow.com/questions/2265503/why-do-i-need-to-override-the-equals-and-hashcode-methods-in-java#:~:text=You%20must%20override%20hashCode(),HashMap%2C%20HashSet%2C%20and%20Hashtable.

Reply:
done